generators/base/src/AbstractGeneratorAgent.ts

```diff
@@ -125,10 +125,24 @@ export abstract class AbstractGeneratorAgent<GeneratorContext extends AbstractGe
         return undefined;
     }
 
-    private getFeaturesConfigPath(): string {
-        if (process.env.NODE_ENV === "test") {
-            return path.join(__dirname, "../../features.yml");
+    private async getFeaturesConfigPath(): Promise<string> {
+        // try to find the features.yml file
+        // in local test mode, the features.yml is relative to the generator
+        for (const each of [
```

Comment:
features.yml can also be at `./assets/features.yml`, which is the case for TypeScript SDK generator.

Reply:
I put in several options, should cover it.